frameworks/Lua/octopus/config.lua

```diff
@@ -0,0 +1,39 @@
+return {
+	extensions = {
+		{octopusExtensionsDir, "core"}, 
+		{octopusExtensionsDir, "baseline"}, 
+		{octopusExtensionsDir, "orm"}, 
+		{octopusExtensionsDir, "app"},
+	},
+	
+	octopusExtensionsDir = octopusExtensionsDir,
+	octopusHostDir = octopusHostDir,
+	port = 8080,
+	securePort = 38080,
+	luaCodeCache = "on",
+	serverName = "localhost",
+	errorLog = "error_log logs/error.log;",
+	accessLog = "access_log logs/access.log;",
+	includeDrop = [[#include drop.conf;]],
+	maxBodySize = "50k",
+	minifyJavaScript = false,
+	minifyCommand = [[java -jar ../yuicompressor-2.4.8.jar %s -o %s]],
+	
+	databaseConnection = {
+		rdbms       =   "mysql",
+		host        =   "127.0.0.1",

```

Comment:
Could you add a line to your setup.sh file that replaces this with the value in `$DBHOST`? On our actual test systems, we have 3 separate machines for the app server, the database, and the client load-generator, so we can't count on them all being `127.0.0.1`.

Here's an [example](https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/Lua/lapis/setup.sh) of what I mean. You can probably just use that sed code and just drop in the appropriate file (config.lua).